components/CountdownContainer.js

```diff
@@ -26,10 +26,25 @@ class CountdownContainer extends React.Component {
 function renderCompleted () {
   return (
     <span>
-      Good luck and have fun.
+      Remaining ticket released!
       <style jsx>{`
         span {
           font-size: 36px;
+          position: relative;
+          bottom: 10px;
+          opacity: 0; 
+          animation: fade-in-from-top 1.5s;
+          animation-fill-mode: forwards;
+        }
+        @keyframes fade-in-from-top {
+          from {
+            opacity: 0;
+            bottom: 20px;
```

Comment:
Please don’t animate the `bottom` property because it causes a layout reflow. Instead, please animate the `transform` property.